.github/workflows/build-python-wheels.yml

```diff
@@ -20,41 +21,52 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-
-      - uses: actions/setup-python@v2
-        name: Install Python
         with:
-          python-version: '3.7'
-
-      - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel
+          # Necessary to fetch tags and allow setuptools_scm
+          # see: https://github.com/pypa/setuptools_scm/issues/480
+          fetch-depth: 0
 
-      - name: Prepare source
-        run: |
-          pip install -e .
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
 
-      - name: Remove built files
-        if: (! contains(matrix.os, 'windows'))
-        run: rm -rf build
+      - name: Set build configuration
+        run: cp githubproject.toml pyproject.toml
 
       - name: Build wheels
-        run: python -m cibuildwheel --output-dir wheelhouse
-        env:
-          # We don't yet support Python 3.10.
-          # Pypy does not have Scipy so we cannot support it.
-          CIBW_SKIP: cp310* pp*
-          CIBW_BUILD_VERBOSITY: 1
-
-      - name: Build sdist
-        run: python setup.py sdist
+        uses: pypa/cibuildwheel@v2.3.1
+        with:
+          package-dir: pyat
 
       - name: Upload wheels
         uses: actions/upload-artifact@v2
         with:
           name: wheels
-          path: ./pyat/wheelhouse/*.whl
+          path: ./wheelhouse/*.whl
           if-no-files-found: error
 
+  build_sdist:
```

Comment:
What is required to fully test the GitHub Actions jobs?

Would you mind pushing the `pyat-test` tag to at least ensure that process runs correctly.

Reply:
I tested the wheels by running the action "on push". Wheels look correct, though I could not test on Windows.

Reply:
How did you run the action 'on push'?

I've looked at the jobs that were run 2 days ago, and none of them have produced any artifacts. I'm looking for something like this: https://github.com/atcollab/at/actions/runs/1282945133

It appears the last time this was run for the `python_packaging` branch was 12 days ago.

Reply:
I just created a temporary "python_pack" branch, changed the "on:" sequence in the .yml file and pushed. Then I downloaded the artifacts and tested them on linux and mac. Apparently all this disappeared when I deleted the branch.

Reply:
Okay, that sounds reasonable.

I think the `pyat-test` tag was made for this purpose, and is probably easier to mess around with.

Reply:
Pushing the tag would have worked only once the branch was merged into master. And when it is, it will now be easier to trigger it from the GitHub interface, no need for a fake tag any more.